<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VqbMainWindow</class>
 <widget class="QMainWindow" name="VqbMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>813</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>VqbForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>757</width>
         <height>486</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="action_Return_to_Startup"/>
   </widget>
   <widget class="QMenu" name="menuWeb">
    <property name="title">
     <string>&amp;Web</string>
    </property>
    <addaction name="actionPostToPastebin"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuWeb"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="baseSize">
    <size>
     <width>779</width>
     <height>248</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <property name="baseSize">
     <size>
      <width>779</width>
      <height>226</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="baseSize">
        <size>
         <width>751</width>
         <height>221</height>
        </size>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabQuery">
        <property name="statusTip">
         <string>Displays current query</string>
        </property>
        <attribute name="title">
         <string>Resulting Query</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="queryViewer">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabResults">
        <property name="statusTip">
         <string>Test-runs the query</string>
        </property>
        <attribute name="title">
         <string>Query Results</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTextEdit" name="queryResults"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPostToPastebin">
   <property name="text">
    <string>Post to pastebin.com</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Return_to_Startup">
   <property name="text">
    <string>&amp;Return to Startup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>queryViewer</tabstop>
  <tabstop>queryResults</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
